src/components/Dropdown/Dropdown.tsx

```diff
@@ -34,6 +34,8 @@ export class Dropdown extends React.PureComponent<Props, State> {
   private getUniqueID = createUniqueIDFactory('Dropdown');
   private activatorContainer: HTMLElement | null;
   private id = this.getUniqueID();
+  private dropdownEle: any = null;
```

Comment:
@dheeraja00 As mentioned before, we want to write out entire names: `dropdownElement`  Also, we want to avoid `: any`.  A simple google search returned https://stackoverflow.com/questions/14742194/declaring-an-htmlelement-typescript and `HTMLElement is the type we can use.`

Reply:
@harsimranb sure I will avoid use of `any` anywhere, and will make changes to every places where I have used it